### ISSUE TITLE
Failure to extract archive - 'dict' object has no attribute 'split'

### ISSUE THREAD

**Issue by tumbl3w33d** [borgbase/vorta#1010]
**Describe the bug

When trying to extract a backup, an error message pops up.
```
Uncaught exception, please file a report with this text at
https://github.com/borgbase/vorta/issues/new
Traceback (most recent call last):
  File "/app/lib/python3.7/site-packages/vorta/views/archive_tab.py", line 416, in list_archive_result
    window = ExtractDialog(result['data'], archive)
  File "/app/lib/python3.7/site-packages/vorta/views/extract_dialog.py", line 42, in __init__
    for line in fs_data.split("\n"):
AttributeError: 'dict' object has no attribute 'split'
```

**To Reproduce**
1. Go to `Archives`
2. Click on the latest (on top of the list) archive in the list
3. Click on `Selected Archive`
4. Choose `Extract`
5. See error

**Environment (please complete the following information):**
- OS: `5.10.41-1-MANJARO #1 SMP PREEMPT Fri May 28 19:10:32 UTC 2021`
- Vorta version: 0.7.5
- Borg version: 1.1.15
- Installed from: Flatpak (flathub)

Vorta and Borg versions can be found in Main Window > Misc Tab.

**Additional context**
```
2021-06-08 12:43:41,475 - asyncio - DEBUG - Using selector: EpollSelector
2021-06-08 12:43:41,476 - vorta.borg.borg_thread - DEBUG - Using VortaSecretStorageKeyring keyring to store passwords.
2021-06-08 12:43:41,477 - asyncio - DEBUG - Using selector: EpollSelector
2021-06-08 12:43:41,478 - root - DEBUG - Found 1 passwords matching repo URL.
2021-06-08 12:43:41,517 - vorta.borg.borg_thread - INFO - Running command /app/bin/borg list --info --log-json --json-lines --format {size:8d}{TAB}{mtime}{TAB}{path}{NL} /home/redacted/backup::redacted-2021-06-07-200000
2021-06-08 12:43:42,090 - root - CRITICAL - Uncaught exception, file a report at https://github.com/borgbase/vorta/issues/new
Traceback (most recent call last):
  File "/app/lib/python3.7/site-packages/vorta/views/archive_tab.py", line 416, in list_archive_result
    window = ExtractDialog(result['data'], archive)
  File "/app/lib/python3.7/site-packages/vorta/views/extract_dialog.py", line 42, in __init__
    for line in fs_data.split("\n"):
AttributeError: 'dict' object has no attribute 'split'
```
I've executed the mentioned command in the flatpak sandbox manually and the result was:
```
{"type": "d", "mode": "drwxr-xr-x", "user": "nfsnobody", "group": "nfsnobody", "uid": 65534, "gid": 65534, "path": "mnt/nextcloud", "healthy": true, "source": "", "linktarget": "", "flags": null, "size": 0, "mtime": "2021-06-06T13:04:02.230021"}
```
Now the question is, why a string split is called on a dictionary.

**Comment by bastiencyr** [comment]
Hello, 
It seems to be fix on #910 . You should not have this issue on current master branch.

**Comment by m3nu** [comment]
Correct. Already in `master` and I'm just in the middle of releasing v0.7.6, which will have the fix.

If the issue still happens with this very latest release, please comment again here.
